HoloPipelines/pipelineConfig.py

```diff
@@ -0,0 +1,140 @@
+import json
+import os
+import sys
+import argparse
+import subprocess
+import pathlib
+import logging
+import importlib
+from pipelines.components.compGetPipelineListInfo import get_pipeline_list
+
+from multiprocessing import Process
+import pipelines.components.compCommonPath as pl_common_path
+
+
+FORMAT = "%(asctime)-15s -function name:%(funcName)s -%(message)s"
+logging.basicConfig(level=logging.INFO, format=FORMAT)
+
+
+new_cwd = str(pathlib.Path(str(os.path.dirname(os.path.realpath(__file__)))))
+
+
+parser = argparse.ArgumentParser(description="Select pipeline to process")
+parser.add_argument(
+    "-c",
+    "--config",
+    default="pipelineList.json",
+    type=str,
+    help="path to pipeline config file relative to pipelineConfig",
+)
```

Comment:
Duplicate Code

It's currently on:
HoloPipelines\pipelineConfig.py:29-82
HoloPipelines\pipelineController.py:26-79

Reply:
I'll fix it in refactoring branch.